client/src/Molecules/RocketWithPlanets/RocketWithPlanetsStyled.js

```diff
@@ -0,0 +1,19 @@
+import planets from '../../Assets/planets.png';
+import styled from 'styled-components';
+
+const RocketWithPlanetsStyled = styled.div`
+  @media only screen and (min-width: 1600px) {
```

Comment:
It shows up on smaller screens for me, you could add `  display: none;
` between lines 4 and five not to show on smaller than 1600. Check the display for 1601px, the rocket is not visible 
![image](https://user-images.githubusercontent.com/58532638/158020797-7236be9e-eba3-4c86-bf95-cbc4aef79181.png)